.github/workflows/zeebe-ci.yml

```diff
@@ -128,12 +131,10 @@ jobs:
           secret_vault_roleId: ${{ secrets.VAULT_ROLE_ID }}
   unit-tests:
     name: Unit tests
-    runs-on: [ self-hosted, linux, amd64, "16" ]
+    runs-on: gcp-perf-core-16-default
     timeout-minutes: 30
     steps:
       - uses: actions/checkout@v4
-      - name: Install strace tests
-        run: sudo apt-get -qq update && sudo apt-get install -y strace
```

Comment:
:question: So we backported already the other changes?

Reply:
Strace tests are disabled in `stable/8.3`, see https://github.com/camunda/camunda/blob/stable/8.3/snapshot/src/test/java/io/camunda/zeebe/snapshots/impl/SnapshotChecksumTest.java#L133` annotation.
This step is not necessary.